content/master/concepts/pods.md

```diff
@@ -0,0 +1,231 @@
+for Crossplane and Crossplane Providers. 
+
+{{<hint "note" >}}
+Crossplane installs and enables the RBAC manager by default. You can disable the
+RBAC manager pod.  
+Disabling the RBAC manager requires manual Kubernetes permissions definitions
+for proper Crossplane operations. 
+
+The 
+[RBAC manager design document](https://github.com/crossplane/crossplane/blob/master/design/design-doc-rbac-manager.md) 
+provides more comprehensive details on the Crossplane RBAC requirements.
+{{< /hint >}}
+
+<!-- vale Microsoft.HeadingAcronyms = NO -->
+<!-- allow 'RBAC' since that's the name -->
+### RBAC init pod
+<!-- vale Microsoft.HeadingAcronyms = YES -->
+
+The RBAC manager requires the `CompositeResourceDefinition` and
+`ProviderRevision` resources to be available before starting.
+
+The RBAC manager init pod waits for these resources before starting the main 
+RBAC manager pod. 
+
+
+<!-- vale Microsoft.HeadingAcronyms = NO -->
+<!-- allow 'RBAC' since that's the name -->
+### RBAC manager pod
+<!-- vale Microsoft.HeadingAcronyms = YES -->
+
+The RBAC manager pod preforms the following tasks:
+* creating and applying RBAC policies to Provider Service Accounts, allowing 
+  them to control their managed resources
+* allowing the `crossplane` Service Account to create managed resources
+* creating and applying policies for Crossplane Composite Resources
+
+When installing a Provider their service account needs permissions to create and 
+manage managed resources. 
+
+When creating Composite Resources the Crossplane service account needs
+permissions for the newly created API endpoints. 
+
+The RBAC manager pod manages these functions. 
+
+The RBAC manager pod supports two policy options:
+* `All` - The default policy, configures and manages all RBAC capabilities.
+* `Basic` - Only manage policies related to Composite resources. 
+
+Change from the default `All` to the `Basic` policy with the `-m Basic` pod 
+argument.
+
+Disable the RBAC manager after installation by deleting the
+`crossplane-rbac-manager` deployment from the `crossplane-system` namespace.
+
+Disable the RBAC manager before installation by editing the Helm `values.yaml`
+file, setting `rbacManager.deploy` to `false`.
+
+{{< hint "note" >}}
+
+Instructions for changing Crossplane pod settings during installation are in the
+[Crossplane Install]({{<ref "../software/install">}}) section. 
+{{< /hint >}}
+
+## Leader election
+
+By default only a single Crossplane pod runs in a cluster. If more than one
+Crossplane pod runs both pods try to manage Crossplane resources. To prevent
+conflicts Crossplane uses a _leader election_ to have a single pod in control at
+a time. Other Crossplane pods standby until the leader fails. 
+
+{{<hint "important" >}}
+Leader election applies to both the Crossplane pod and the RBAC manager pod.
+{{< /hint >}}
+
+Enable leader elections with the `--leader-election` pod argument.
```

Comment:
Would this argument set which XP pod becomes the leader?

Reply:
No - it is random based on timing.  It is even possible that the "failed" pod could regain the leader role after it restarts if the timer for the "standby" pod(s) has not expired yet.

Reply:
No, "who" is the leader doesn't matter since it's just another pod in the deployment. Just like the specific node a pod runs on doesn't matter. 

The election itself is driven by kubernetes. 

Should I add more context?